src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString.split(';');
+  const cssObject = {};
+  const objectEntries= [];
+  cssProperties.forEach(element => objectEntries.push(element.split(':')));
```

Comment:
Для таких ситуаций есть метод map.
```suggestion
  objectEntries = cssProperties.map(...*your code*)
```